security/1.0/apidocs/allpackages-index.html

```diff
@@ -0,0 +1,153 @@
+</li>
+</ul>
+</div>
+<a id="skip.navbar.top">
+<!--   -->
+</a>
+<!-- ========= END OF TOP NAVBAR ========= -->
+</div>
+<div class="navPadding">&nbsp;</div>
+<script type="text/javascript"><!--
+$('.navPadding').css('padding-top', $('.fixedNav').css("height"));
+//-->
+</script>
+</nav>
+</header>
+<main role="main">
+<div class="header">
+<h1 title="All&amp;nbsp;Packages" class="title">All&nbsp;Packages</h1>
+</div>
+<div class="allPackagesContainer">
+<ul class="blockList">
+<li class="blockList">
+<div class="packagesSummary">
+<table>
+<caption><span>Package Summary</span><span class="tabEnd">&nbsp;</span></caption>
+<tr>
+<th class="colFirst" scope="col">Package</th>
+<th class="colLast" scope="col">Description</th>
+</tr>
+<tbody>
+<tr class="altColor">
+<th class="colFirst" scope="row"><a href="javax/security/enterprise/package-summary.html">javax.security.enterprise</a></th>
+<td class="colLast">
+<div class="block">The main Jakarta Security package.</div>
+</td>
+</tr>
+<tr class="rowColor">
+<th class="colFirst" scope="row"><a href="javax/security/enterprise/authentication/mechanism/http/package-summary.html">javax.security.enterprise.authentication.mechanism.http</a></th>
+<td class="colLast">
+<div class="block">The HTTP authentication mechanism API package.</div>
+</td>
+</tr>
+<tr class="altColor">
+<th class="colFirst" scope="row"><a href="javax/security/enterprise/credential/package-summary.html">javax.security.enterprise.credential</a></th>
+<td class="colLast">
+<div class="block">The Identity Store Credential API package.</div>
+</td>
+</tr>
+<tr class="rowColor">
+<th class="colFirst" scope="row"><a href="javax/security/enterprise/identitystore/package-summary.html">javax.security.enterprise.identitystore</a></th>
+<td class="colLast">
+<div class="block">The identity store API package.</div>
+</td>
+</tr>
+</tbody>
+</table>
+</div>
+</li>
+</ul>
+</div>
+</main>
+<footer role="contentinfo">
+<nav role="navigation">
+<!-- ======= START OF BOTTOM NAVBAR ====== -->
+<div class="bottomNav"><a id="navbar.bottom">
+<!--   -->
+</a>
+<div class="skipNav"><a href="#skip.navbar.bottom" title="Skip navigation links">Skip navigation links</a></div>
+<a id="navbar.bottom.firstrow">
+<!--   -->
+</a>
+<ul class="navList" title="Navigation">
+<li><a href="index.html">Overview</a></li>
+<li>Package</li>
+<li>Class</li>
+<li>Use</li>
+<li><a href="overview-tree.html">Tree</a></li>
+<li><a href="deprecated-list.html">Deprecated</a></li>
+<li><a href="index-files/index-1.html">Index</a></li>
+<li><a href="help-doc.html">Help</a></li>
+</ul>
+</div>
+<a id="skip.navbar.bottom">
+<!--   -->
+</a>
+<!-- ======== END OF BOTTOM NAVBAR ======= -->
+</nav>
+<p class="legalCopy"><small>Copyright &#169; 2015-2017,     <a href="http://www.oracle.com">Oracle</a>     and/or its affiliates. All Rights Reserved.     Use is subject to     <a href="./doc-files/speclicense.html" target="_top">license terms</a>.</small></p>
```

Comment:
Hi, on line 150 I think the correct one should be "Eclipse Foundation" instead of "Oracle", right?

Reply:
If I'm not mistaken copyrights remain at Oracle, but ping @bshannon for confirmation.

Reply:
I [clarified this recently](https://www.eclipse.org/lists/jakartaee-spec-project-leads/msg00163.html).  It needs to be Eclipse Foundation.

Please tell me you're subscribed to the [jakartaee-spec-project-leads mailing list](https://accounts.eclipse.org/mailing-list/jakartaee-spec-project-leads) and are reading all the messages!  :-)

Reply:
Reading all the messages, but sometimes there's a backlog ;) But have been updating all the previous PRs and restaging.